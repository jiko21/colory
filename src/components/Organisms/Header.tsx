import React from 'react';
import { Flex, Header as SpectrumHeader, View } from '@adobe/react-spectrum';

const Header: React.FC = ({ children }) => {
  return (
    <SpectrumHeader>
      <Flex direction="row" gap="size-400" alignItems="center" justifyContent="center">
        <View>{children}</View>
      </Flex>
    </SpectrumHeader>
  );
};

export default Header;
