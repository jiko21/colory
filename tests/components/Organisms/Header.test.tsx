import Header from '@/components/Organisms/Header';
import { render } from '@testing-library/react';
import React from 'react';

describe('components/Organisms/Header.tsx', () => {
  it('correctly render', () => {
    const component = render(<Header>Sample</Header>);
    expect(component.container).toMatchSnapshot();
  });
});
