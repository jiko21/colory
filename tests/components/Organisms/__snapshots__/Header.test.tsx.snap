// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Organisms/Header.tsx correctly render 1`] = `
<div>
  <header>
    <div
      class="_flex-container_ffbb7"
    >
      <div
        class="_flex_ffbb7 _flex-gap_ffbb7"
        style="flex-direction: row; align-items: center; justify-content: center; --gap: var(--spectrum-global-dimension-size-400, var(--spectrum-alias-size-400));"
      >
        <div>
          Sample
        </div>
      </div>
    </div>
  </header>
</div>
`;
